pyapp/cookiecutter.json

```diff
@@ -12,8 +12,7 @@
     "dependabot_npm_interval": ["monthly", "weekly", "daily"],
     "dependabot_docker_interval": ["monthly", "weekly", "daily"],
     "devdata": ["no", "yes"],
-    "services": ["no", "yes"],
-    "db": ["no", "yes"],
+    "postgres": ["no", "yes"],
```

Comment:
I removed the `services` option. It doesn't make sense for a project to say "Yes, I want services." The project actually has to say _which_ services it wants, otherwise the cookiecutter doesn't know which services to put in `docker-compose.yml` and `ci.yml`.

So instead of the `services` option there are now `docker-compose/services.yml` and `.github/workflows/ci/services.yml` includes. Rather than a boolean option, the existence of these include files is what triggers the `docker-compose.yml` file and the `services` section in `ci.yml` to be generated.

I also renamed the `"db"` option to `"postgres"`, seems like a better name.